src/arrayMethodSort.js

```diff
@@ -4,8 +4,38 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  const stringsCompare = (a, b) => {
+    const stringA = a.toString();
+    const stringB = b.toString();
+
+    if (stringA > stringB) {
+      return 1;
+    } else if (stringB > stringA) {
+      return -1;
+    } else {
```

Comment:
```suggestion
    } 
```

No need for this 'else' because it will work anyway if no of the previous checks won't pass.